qt/bundle.h

```diff
@@ -40,6 +40,11 @@ class APPSTREAMQT_EXPORT Bundle {
         Bundle& operator=(const Bundle& bundle);
         bool operator==(const Bundle& r) const;
 
+        /**
+         * \returns the internally stored AsBundle
+         */
+        _AsBundle *asBundle() const;
```

Comment:
I use these internally, i.e. in as_component_add_icon() or as_component_add_screenshot() and so on, because those methods takes AsIcon or AsScreenshot only.